content/docs/command-reference/commit.md

```diff
@@ -132,7 +131,7 @@ $ dvc pull -aT
 Sometimes we want to iterate through multiple changes to configuration, code, or
 data, trying different ways to improve the output of a stage. To avoid filling
 the <abbr>cache</abbr> with undesired intermediate results, we can run a single
-stage with `dvc run --no-commit`, or reproduce an entire pipeline using
+stage with `dvc exp run --no-commit`, or reproduce an entire pipeline using
```

Comment:
Idk. This produces an experiment which also fills the cache. Should be `repro` I think.

Reply:
Which cache? Run cache or the file cache? `dvc exp run` command works similar to repro, commits artifacts to `.dvc/cache`

Reply:
Regardless of the cache question, `exp run` produces an experiment. We have not decided to use experiment runs as the default pipeline workflow that I know of so any non-experiment specific docs should not use exp commands. Thus `repro` here.

Reply:
Agreed, this is not a valid change semantically.

Probably `dvc stage add` and then `dvc repro --no-commit <stage>` is the way to replace it.

Or ideally `dvc stage add` should have `--execute`.

Reply:
> Probably dvc stage add and then dvc repro --no-commit <stage> is the way to replace it.

Agree, emphasizing the `<stage>` given as `target`.

> I removed the "entire pipeline" phrase

Why? It's a valid situation we want to mention. If we're going to change that, rather generalize to `repro --no-commit` only and don't mention either specific-stage(s) nor entire-pipeline cases, as that distinction is besides the point, really.

Reply:
Restored the phrase now. PTAL